scripts/run_alpaca_temperature_experiment.py

```diff
@@ -0,0 +1,15 @@
+from inspect_ai import eval_set
+
+from unexploitable_search.tasks.alpaca.past_alpaca import past_alpaca
+
+tasks = [
+    past_alpaca(temp)
```

Comment:
Future work. Although this is a really small script, it'd be nice to see if we can make this modular and make it work for both NL and coding tasks. This is just so we don't repeat code which could lead to issues in the future